postBuildStep.ts

```diff
@@ -140,4 +153,23 @@ function isDirectory(fullPath: string) {
     }
 }
 
+async function patchFileForWebViewWithLegacySupport(

```

Comment:
I think we can remove this function and just assume we always build with the latest version of the tools. We probably wouldn't be using this patching mechanism if we wanted to host older versions of the tools in the webview anyway.